exercises/space-age/README.md

```diff
@@ -63,14 +61,14 @@ No compiler found, expected minor version match with...
 Try running "stack setup" to install the correct GHC...
 ```
 
-Just do as it says and it will download and install
+Just do as it says, and it will download and install
 the correct compiler version:
 
 ```bash
 stack setup
 ```
 
-## Running *GHCi*
+## Running _GHCi_
```

Comment:
This is actually a change to the template file, [config/exercise_readme.go.tmpl](https://github.com/exercism/haskell/blob/master/config/exercise_readme.go.tmpl).

I would also ask here: does this conform to some particular Markdown standard?

Reply:
I was wrong about this, checking the official syntax. The best practice use is the asterisks for compatibility reasons. I will update this in the correct location and revert back to the old syntax.